src/reporters/lang/en.js

```diff
@@ -72,6 +72,10 @@ const messages = {
   unexpectedError: 'An unexpected error occurred, please open a bug report with the information provided in $0.',
   jsonError: 'Error parsing JSON at $0, $1.',
 
+  yarnOutdated: "Your current version of Yarn is out of date. The latest version is $0 while you're on $1.",
+  yarnOutdatedInstaller: 'To upgrade, download the latest installer at $0.',
+  yarnOutdatedCommand: 'To upgrade, run $0.',
```

Comment:
Let's put the command on its own separate line. @sindresorhus said:
> Having the command on a separate line makes it easier to copy-paste for users.
https://github.com/yarnpkg/yarn/issues/1138#issuecomment-254114619